Valet.xcodeproj/project.pbxproj

```diff
@@ -211,20 +183,6 @@
 			remoteGlobalIDString = 1612FDD222A9CB2200FC1142;
 			remoteInfo = "LegacyValet iOS";
 		};
-		1612FDEE22A9CB4800FC1142 /* PBXContainerItemProxy */ = {
-			isa = PBXContainerItemProxy;
-			containerPortal = EA1E1F7B1A8C46080067C991 /* Project object */;
-			proxyType = 1;
-			remoteGlobalIDString = 1612FDE322A9CB3100FC1142;
-			remoteInfo = "LegacyValet watchOS";
```

Comment:
`watchOS` support was added in Valet 3. `LegacyValet` is code from Valet 2. This target should never have existed.